### ISSUE TITLE
ion-select with multiple value not work properly on ios devices

### ISSUE THREAD

**Issue by ArtysT** [ionic-team/ionic#12420]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ x ] **3.x**

**I'm submitting a ...**  (check one with "x")
[ x ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

On ios device after selecting option in multichoice select for a fraction of a second it sets the option value to true and then returns false. The problem only occurs on the iPhone. Android and emulator works correctly.

DOM At the moment of clicking:
```
<button class="alert-tappable alert-checkbox disable-hover alert-checkbox-button alert-checkbox-button-ios alert-checkbox-button-default alert-checkbox-button-default-ios activated" ion-button="alert-checkbox-button" role="checkbox" aria-checked="true" id="alert-input-2-0">...</button>
```

DOM After click:
```
<button class="alert-tappable alert-checkbox disable-hover alert-checkbox-button alert-checkbox-button-ios alert-checkbox-button-default alert-checkbox-button-default-ios" ion-button="alert-checkbox-button" role="checkbox" aria-checked="false" id="alert-input-2-0">...</button>
```

**Related code:**

```
<ion-item *ngFor="let question of questions">
    <ion-label>{{question.question}}</ion-label>
    <ion-select [multiple]="question.questionType == 'CHECKBOX'"
                [formControlName]="'question' + question.questionId">
        <ion-option *ngFor="let answer of question.answers" [value]="answer.value">
           {{answer.answer}}
        </ion-option>
    </ion-select>
</ion-item>
```

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
global packages:

   @ionic/cli-utils : 1.5.0
   Cordova CLI      : 6.5.0 
   Ionic CLI        : 3.5.0

local packages:

   @ionic/app-scripts              : 1.3.8
   @ionic/cli-plugin-cordova       : 1.4.1
   @ionic/cli-plugin-ionic-angular : 1.3.2
   Cordova Platforms               : ios 4.4.0
   Ionic Framework                 : ionic-angular 3.4.2

System:

   Node       : v6.9.0
   OS         : macOS Sierra
   Xcode      : Xcode 8.3.3 Build version 8E3004b 
   ios-deploy : 1.9.1 
   ios-sim    : 5.0.13 
   npm        : 3.10.8
```

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! Can you install our [wkwebview plugin](https://github.com/ionic-team/cordova-plugin-wkwebview-engine) and give this another try please?

**Comment by Angela94** [comment]
I use  [apache wkwebview](https://github.com/apache/cordova-plugin-wkwebview-engine) because only on this I have the ability to send requests to the sites and multiple select now works, but ther is something wrong with getting custome header value from response:

```
    public doLogin(user: any, success: Function, fail: Function) {
        this.http.post(PARSE_SERVER_USER_URL + 'api/auth/login', {
            'username': user.username,
            'password': user.password
        }).subscribe(
            response => {
                if (response.headers) {
                    let token = response.headers.get('x-auth-token');
                    let that = this;
                    this.setUser(response.json()).then(function () {
                        that.loginSuccess(token, response.json(), success);
                    });
                }
            },
            error => {
                this.loginFailure(fail);
            }
        );
    }
```
token is null

I have to investigate it more deeply, but maybe someone knows some possible cause.

**Comment by ArtysT** [comment]
I use  [apache wkwebview](https://github.com/apache/cordova-plugin-wkwebview-engine) (because only on this I have the ability to send requests to the service) and multiple select now work, but ther is something wrong with getting custome header value from response:

```
    public doLogin(user: any, success: Function, fail: Function) {
        this.http.post(PARSE_SERVER_USER_URL + 'api/auth/login', {
            'username': user.username,
            'password': user.password
        }).subscribe(
            response => {
                if (response.headers) {
                    let token = response.headers.get('x-auth-token');
                    let that = this;
                    this.setUser(response.json()).then(function () {
                        that.loginSuccess(token, response.json(), success);
                    });
                }
            },
            error => {
                this.loginFailure(fail);
            }
        );
    }
```
token is null

I have to investigate it more deeply, but maybe someone knows some possible cause.

**Comment by jgw96** [comment]
No problem! The wkwebview plugin will be installed by default in a future release.

**Comment by aaronneon2** [comment]
Is there other ways to fix this issue? Because I tried wkwebview plugin and it produces 2 new other issues just so that I can fix this 1 issue, which doesn't make sense to me.

The new other issues are:
- CORS. The older webview uses to contact my XML-RPC backend server API with no origin specified. It worked until I install wkwebview and it ran into CORS issue on iOS. I had to allow http://localhost:8080 on my servers to allow API access, which to me is a security threat.
- Bouncing of view upon input focus. The lower the input textfield in a page, the more bounce. No configuration seems to be available to fix this issue.
